tests/unit_tests/test_matchmaker_queue.py

```diff
@@ -436,8 +436,8 @@ def make_matches(*args):
         return []
 
     with mock.patch(
-        "server.matchmaker.matchmaker_queue.make_matches",
-        make_matches
+        "server.matchmaker.matchmaker_queue.StableMarriageMatchmaker.find",
```

Comment:
This lookup path is causing the tests to fail. I think this can now be done a lot easier by mocking the `matchmaker` object directly:

```python
queue.matchmaker.find = mock.Mock(side_effect=find)
```
something like that